doozer/doozerlib/git_helper.py

```diff
@@ -0,0 +1,66 @@
+import logging
```

Comment:
This module could be useful tool wide? Maybe it could be part of `artcommonlib`

Reply:
This is a standard library. https://github.com/openshift-eng/art-tools/pull/725 makes it possible to log with standard logging library to print logs to both stderr and debug.log. I would prefer to directly use standard library until we find a use case that certain code pattern is reusable.

Reply:
I was actually referring to the `git_helper` module. Since this seems to offer general purpose functionalities, we might benefit from it in other places (e.g. elliott, pyartcd)